selfservice/strategy/code/strategy_login.go

```diff
@@ -303,13 +290,11 @@ func (s *Strategy) findIdentityForIdentifier(ctx context.Context, identifier str
 					return nil, nil, errors.WithStack(schema.NewNoCodeAuthnCredentials())
 				}
 
-				address, err := s.findIdentifierInVerifiableAddress(session.Identity, identifier)
+				_, err := s.findIdentifierInVerifiableAddress(session.Identity, identifier)
 				if err != nil {
 					return nil, nil, err
 				}
 
-				addresses = []Address{*address}
-
```

Comment:
Revert this please

Reply:
Funny how the linter doesn't pick this up properly 👀

Reply:
My bad, it is indeed in-effective.